guide/payments/send.md

```diff
@@ -39,169 +109,184 @@ layout = "full-width"
 
 # Sending bitcoin
 
-Sending bitcoin can be a very straightforward or complex flow in a Bitcoin application. People may be sending bitcoin to a known contact, moving it between their wallets on different devices, or making a purchase through a [payment processor]({{ '/guide/getting-started/software/#payment-processors' | relative_url }}).
+Sending bitcoin is one of the most essential user activities in a Bitcoin application, and also one of the least structured ones. People may want to send bitcoin to a known contact, transfer it to another wallet on a different device, or make a purchase via a [payment processor]({{ '/guide/getting-started/software/#payment-processors' | relative_url }}).
 
-Let us look at the main transaction options senders need to configure when moving bitcoin:
+## Payment entry points
```

Comment:
Another payment entry point you may want to consider adding is generating and sharing a withdraw request. I'm covering this in my requesting bitcoin page. By withdraw request I'm referring to LNURL-withdraw (withdraw request sounds better imo). 

You could expand on this somewhere in this section and discuss how these could be used to generate P2P gift cards.

Reply:
I was thinking we could have a "Withdrawing bitcoin" page at a later point.

Reply:
Would there be enough content for another dedicated page? Could easily be covered across sending / request page imo

Reply:
I don't think it would be enough for a dedicated page. It's def a use-case we should cover, perhaps in a follow up? One more example besides a gift would be a user getting a refund from a merchant so they need to scan a QR in order to pull the funds out.